File: app/src/main/java/fup/prototype/robprototype/view/bindings/Bindings.java
package fup.prototype.robprototype.view.bindings;

import android.databinding.BindingAdapter;
import android.view.View;

public class Bindings {

    @BindingAdapter({"visibleOrGone"})
    public static void setVisibleOrGone(final View view, final boolean isVisible) {
        view.setVisibility(isVisible ? View.VISIBLE : View.GONE);
    }

}

File: app/src/main/java/fup/prototype/robprototype/view/bindings/Bindings.kt
package fup.prototype.robprototype.view.bindings

import android.databinding.BindingAdapter
import android.view.View
import android.widget.ImageView
import com.bumptech.glide.Glide

object Bindings {

    @JvmStatic
    @BindingAdapter("visibleOrGone")
    fun setVisibleOrGone(view: View, isVisible: Boolean) {
        view.visibility = if (isVisible) {
            View.VISIBLE
        } else {
            View.GONE
        }
    }

    @JvmStatic
    @BindingAdapter("glideAvatarUrl")
    fun setGlideAvatarUrl(view: ImageView, url: String) {
        Glide.with(view.context)
                .asDrawable()
                .load(url)
                .into(view)
    }

}